google/data_source_google_compute_region_instance_group_manager.go

```diff
@@ -0,0 +1,161 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	compute "google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+	"log"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func dataSourceGoogleComputeRegionInstanceGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceComputeRegionInstanceGroupRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"items": {
```

Comment:
We tend to not pluralize field names (no always followed). Would that make more sense to call it `instance` instead of `item`?

Reply:
No strong feelings - I thought it would be best to stick to exactly the field names that come back from the /listInstances call.  Do you prefer `instance` to `items` given that `items` is what's used here?
https://cloud.google.com/compute/docs/reference/latest/regionInstanceGroups/listInstances

FWIW I agree that "items" is a lousy name!  It's probably the least specific you can be about what's in the list.

Reply:
`instance` is definitely better. I think actually the pluralized version would be better since it is read only.

people will use it for interpolation in their data source this way:
```hcl
instance = "${data.google_compute_region_instance_group_manager.instances.0.self_link}"
```

Reply:
Agreed, and done.